en/docs/develop/customizing-user-portal-ui.md

```diff
@@ -0,0 +1,210 @@
+    `modules/theme/src/themes/default/assets/images` folder(owl
+    .svg).
+
+    2.  In `modules/theme/src/definitions/globals/product.less` replace the existing styles in .product-logo
+    class with the following.
+        ```java
+            .product-title {
+                .product-logo {
+                    width: 25px;
+                    height: 25px;
+                    vertical-align: text-top;
+                    margin-right: 5px;
+                    background: url(assets/images/owl.svg) no-repeat;
+                    background-size: auto;
+
+                    svg {
+                        display: none;
+                    }
+                }
+
+                // Other styles
+            }
+        ```
+
+    ### Method 2
+
+    1. Add the downloaded icon to the `modules/theme/src/themes/default/assets/images` folder.
+    2. In modules/theme/src/index.js replace Logo with the path to the new icon.
+            ```java
+                export const Logo = require("../lib/assets/images/owl.svg");
+            ```
+    3.  Build user-portal artifacts.
+            ```java
+                npx lerna run build — scope @wso2is/user-portal
+            ```
+    4. Replace **main.js** & **main.js.map** inside the **user-portal** web app with the same from
+        `apps/user-portal/build/user-portal`.
+
+2.  Change the product title & Copyright.
+
+    In the IS pack, the **User Portal** web app is available in the path
+    `<IS_HOME>/repository/deployment/server/webapps/user-portal`
+    In index.jsp add the following two entries in the runConfig window object.
+    ```java
+        window["runConfig"] = {
+            ...
+            applicationName: "NIGHT OWL EXPRESS",
+            copyrightText: "Night Owl Express © 2020"
+        };
+    ```
+
+3.  Change the Favicon and Title.
+
+    Replace favicon.ico at the root of the user-portal web app with the desired icon. You can use an online generator
+     like [favicon.io](https://favicon.io/) to generate a favicon for free.
+
+    To change the title, in **index.jsp** file of the web app, change the `<title>` tag.
+    ```html
+        <title>Night Owl Express</title>
+    ```
+
+### Step 3: Deploying the changes in the web app
+
+We are at the final step of the process which is the deployment. Follow the sequence of steps listed below to deploy
+the changes performed in the previous steps.
+
+1.  Build the theme module.
+```java
+    # from inside modules/theme
+    $npm run build
+```
+2.  Copy the artifacts to the web app.
+
+    The built artifacts will be available inside the **lib** folder. Copy everything to the clipboard and navigate to
+     the **user-portal** web app in the IS pack.
+    user-portal web app is available in the location `<IS_HOME>/repository/deployment/server/webapps/user-portal`
+
+    Replace the contents inside `/libs/styles/css` with the copied artifacts.
```

Comment:
Copy everything to the clipboard and paste it inside the `<IS_HOME>/repository/deployment/server/webapps/user-portal/libs/styles/css` folder.